apps/payment/constants.py

```diff
@@ -0,0 +1,111 @@
+# Make sure these exist in settings if they are to be used.
```

Comment:
I'd reword the comment to say "These settings are configured here: settings/file.py".

Reply:
Sure, I just copied this from the original location in the Payment model :sweat_smile: